src/styles/mixins/normalize.less

```diff
@@ -0,0 +1,453 @@
+/*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */
+#mixin-Normalize {
```

Comment:
Mixin is already implied by the use of a less namespace, also, could you make this non capitalized? It conflicts with an existing convention.